bindings/java/src/main/native/jucx_common_def.h

```diff
@@ -56,6 +57,9 @@ bool j2cInetSockAddr(JNIEnv *env, jobject sock_addr, sockaddr_storage& ss, sockl
 struct jucx_context {
     jobject callback;
     volatile jobject jucx_request;
+    bool is_completed;
+    ucs_status_t status;
```

Comment:
can we reuse status to indicate if completed. for example status==UCS_INPROGRESS iff "not completed"